files/swap.sh

```diff
@@ -28,13 +29,19 @@ then
   fi
 fi
 
+if [ $MKSWAP_MIN -gt $SWAP_TO_CREATE ]; then
+  SWAP_TO_CREATE=$MKSWAP_MIN
+fi
+
+
 if [ "$SWAP" == "none" ]
 then
   exit 0
 else
-  TMP=`mktemp`
+  TMP='/var/oracle-xe.swapfile'
```

Comment:
My bad, i forgot that.

I change that for two reasons:
 * /tmp files are erased on reboot[1] on Centos 7
 * the better way to solve that problem its creating a file out it and add one line at the `/etc/fstab` file, something like this:
```
/var/oracle-xe.swapfile   swap    swap    sw  0   0
```

I found a easy way to extend swap on centos 7, take a look[2].

[1] http://wiki.centos.org/TipsAndTricks/TmpOnTmpfs
[2] https://www.digitalocean.com/community/tutorials/how-to-add-swap-on-centos-7